file_organizer_v2/src/file_organizer/cli/dedupe.py

```
@@ -0,0 +1,610 @@
+    # Display configuration
+    config_text = (
+        f"[bold]Directory:[/bold] {config.directory}\n"
+        f"[bold]Algorithm:[/bold] {config.algorithm.upper()}\n"
+        f"[bold]Strategy:[/bold] {config.strategy}\n"
+        f"[bold]Recursive:[/bold] {'Yes' if config.recursive else 'No'}\n"
+        f"[bold]Safe Mode:[/bold] {'Enabled' if config.safe_mode else 'Disabled'}\n"
+        f"[bold]Mode:[/bold] {'DRY RUN' if config.dry_run else 'LIVE'}"
+    )
+
+    if config.batch and config.strategy != "manual":
+        config_text += f"\n[bold]Batch Mode:[/bold] Enabled (auto-apply strategy)"
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)